src/plugins/intel_gpu/tests/test_cases/activation_simple_gpu_test.cpp

```diff
@@ -1683,12 +1689,7 @@ const auto reluParams = testing::ValuesIn(std::vector<activation_random_test_par
 INSTANTIATE_TEST_SUITE_P(relu_activation_blocked_tests, activation_random_test, reluParams);
 
 const std::vector<data_types> dataTypes = {data_types::f16, data_types::f32};
-const std::vector<format::type> types = {format::bfyx,
-                                         format::bfzyx,
-                                         format::yxfb,
-                                         format::byxf,
-                                         format::fyxb,
-                                         format::b_fs_yx_fsv2,
+const std::vector<format::type> types = {format::b_fs_yx_fsv2,
```

Comment:
Let's not narrow types. formats like bfzyx requires validation..

Reply:
https://github.com/openvinotoolkit/openvino/pull/12667/files#r954371442

Reply:
Also, Isn't this test added to validate blocked format? It seems that bfzyx has some testcases.